server/src/api/v2/teams/delete-team.ts

```diff
@@ -1,13 +1,16 @@
-import {Request, Response} from 'express';
-import {wrapResponse} from '../../../functions/response-wrapper';
-import {Team} from '../../../models/team.model';
-import {Membership} from '../../../models/membership.model';
+import { Request, Response } from 'express';
+import { wrapResponse } from '../../../functions/response-wrapper';
+import { Team } from '../../../models/team.model';
+import { Membership } from '../../../models/membership.model';
 import { PollAnswer } from '../../../models/poll-answer.model';
 import { Poll } from '../../../models/poll.model';
+//import { Transaction } from 'sequelize';
+//import { Vars } from '../../../vars';
 
 export async function deleteTeam(req: Request, res: Response): Promise<Response> {
     let success = true;
-    //TODO Authoriaztion check, if teams can get created by no-admins
+    // const transaction = new Transaction(Vars.db, {});
```

Comment:
Kommt das in einem extra PR? Oder versehentlich vergessen 😅

Reply:
nicht zum funktionieren gebracht und auf später verschoben ;)